apis/v1alpha2/referencegrant_types.go

```diff
@@ -35,21 +39,15 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // All cross-namespace references in Gateway API (with the exception of cross-namespace
 // Gateway-route attachment) require a ReferenceGrant.
 //
-// Support: Core
+// ReferenceGrant is a form of runtime verification. Implementations that support
```

Comment:
ReferenceGrant is a form of runtime verification _that is asserted by the implementer_ (right?)

Reply:
Possibly? :thinking: 

In #1569 I tried some different language which I hope might help make how these are asserted more clear:

cc80822bc68f6d1c27f9103c837f65cf28208907

Let me know your thoughts?